### PR TITLE
Add --ca-cert arg to encrypt-ami

### PR DESCRIPTION
**Pull request by bonzotronix**
* New suppressed arg for --ca-cert specifies a file to use for the Yeti CA cert. That file contents gets added to the brkt-files part.
* Changed 'description' arg to 'help' arg for one of the add_argument() calls, to prevent an error.

Testing:
* Ran 'encrypt-ami' with and without new arg.
* Ran tests (had to exclude test_gce manually).
* TODO: add unit test or two (held off since the tests are being split up by Boris currently).

Jira: YETI-1062

### CODE REVIEW

File: brkt_cli/aws/__init__.py [hunk 1]
```diff
@@ -351,6 +351,19 @@ def command_encrypt_ami(values, log):
     proxy_config = brkt_cli.get_proxy_config(values)
     jwt = brkt_cli.validate_jwt(values.jwt)
 
+    if values.ca_cert:
+        if not values.brkt_env:
+            raise ValidationError(
+                'Must provide brkt_config when CA cert is provided.'
```
Comment (bburtin) [inline brkt_cli/aws/__init__.py]: "Must specify --brkt-env when using --ca-cert"?

File: brkt_cli/user_data.py [hunk 2]
```diff
@@ -185,6 +188,15 @@ def combine_user_data(brkt_config=None, proxy_config=None, jwt=None):
             BRKT_FILES_CONTENT_TYPE
         )
 
+    if ca_cert:
+        domain = get_domain_from_brkt_config(brkt_config)
+        ca_cert_filename = "/var/brkt/ami_config/ca_cert.pem.%s" % (domain)
+        udc.add_file(
```
Comment (bburtin) [inline brkt_cli/user_data.py]: Do we want to do some simple validation to make sure the file content really is a cert?

File: brkt_cli/aws/__init__.py [hunk 3]
```diff
@@ -351,6 +351,19 @@ def command_encrypt_ami(values, log):
...
+            )
+        try:
+            with open(values.ca_cert, 'r') as f:
+                ca_cert_data = f.read()
+        except IOError as e:
```
Comment (bburtin) [inline brkt_cli/aws/__init__.py]: Do we want to do some simple validation to make sure the file content really is a cert?

File: brkt_cli/util.py [hunk 4]
```diff
@@ -122,6 +122,26 @@ def add_brkt_env_to_brkt_config(brkt_env, brkt_config):
         brkt_config['brkt']['api_host'] = api_host_port
         brkt_config['brkt']['hsmproxy_host'] = hsmproxy_host_port
 
+def get_domain_from_brkt_config(brkt_config):
```
Comment (bburtin) [inline brkt_cli/util.py]: Should this be in user_data?  Doesn't seem generally useful.

**Comment by bonzotronix** [comment]
I'll make the requested changes in a new PR.

**Pull request closed without merging**